coord/src/dist_aggregate.c

```diff
@@ -64,15 +87,22 @@ static int netCursorCallback(MRIteratorCallbackCtx *ctx, MRReply *rep, MRCommand
   }
 
   if (bail_out) {
+    if (cmd->rootCommand == C_DEL && MRReply_Type(rep) == MR_REPLY_ERROR) {
+      // Unsuccessful DEL command
+      RedisModule_Log(NULL, "warning", "No index was found for deletion in the shard");
+    } else if (cmd->rootCommand == C_READ) {
+      RedisModule_Log(NULL, "warning", "An empty reply was received from a shard");
```

Comment:
This is not necessarily an empty reply right? Might be a reply of type we did not expected? Maybe we should print more information here?

Reply:
Right. I'll change it to "unexpected reply", and add some logging (above).